src/App/Multisite.php

```diff
@@ -8,15 +7,18 @@
  * @copyright  2014 Ushahidi
  * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
  */
+
+namespace Ushahidi\App;
+
 use League\Url\Url;
 
-class Ushahidi_Multisite
+class Multisite
```

Comment:
This is a much better naming scheme. Dir structure should be a clear, consistent way to traverse the source.